packages/backend/src/graphql/resolvers/asset.ts

```diff
@@ -7,45 +7,50 @@ import {
 } from '../generated/graphql'
 import { Asset } from '../../asset/model'
 import { errorToCode, errorToMessage, isAssetError } from '../../asset/errors'
-import { ApolloContext } from '../../app'
+import { TenantedApolloContext } from '../../app'
 import { getPageInfo } from '../../shared/pagination'
 import { Pagination, SortOrder } from '../../shared/baseModel'
 import { feeToGraphql } from './fee'
 import { Fee, FeeType } from '../../fee/model'
 import { GraphQLError } from 'graphql'
 import { GraphQLErrorCode } from '../errors'
 
-export const getAssets: QueryResolvers<ApolloContext>['assets'] = async (
-  parent,
-  args,
-  ctx
-): Promise<ResolversTypes['AssetsConnection']> => {
-  const assetService = await ctx.container.use('assetService')
-  const { sortOrder, ...pagination } = args
-  const order = sortOrder === 'ASC' ? SortOrder.Asc : SortOrder.Desc
-  const assets = await assetService.getPage(pagination, order)
-  const pageInfo = await getPageInfo({
-    getPage: (pagination: Pagination, sortOrder?: SortOrder) =>
-      assetService.getPage(pagination, sortOrder),
-    page: assets,
-    sortOrder: order
-  })
-  return {
-    pageInfo,
-    edges: assets.map((asset: Asset) => ({
-      cursor: asset.id,
-      node: assetToGraphql(asset)
-    }))
+export const getAssets: QueryResolvers<TenantedApolloContext>['assets'] =
+  async (parent, args, ctx): Promise<ResolversTypes['AssetsConnection']> => {
+    const assetService = await ctx.container.use('assetService')
+    const { sortOrder, ...pagination } = args
+    const order = sortOrder === 'ASC' ? SortOrder.Asc : SortOrder.Desc
+    const assets = await assetService.getPage({
+      pagination,
+      sortOrder: order,
+      tenantId: ctx.tenant.id
+    })
+    const pageInfo = await getPageInfo({
+      getPage: (pagination: Pagination, sortOrder?: SortOrder) =>
+        assetService.getPage({
+          pagination,
+          sortOrder,
```

Comment:
@mkurapov Will you have a look at what I said here:
https://github.com/interledger/rafiki/pull/3152#discussion_r1912053334

We may want to use that mechanism of being able to provide alternative `tenantId` for the operator?

Reply:
Might even be worth allowing no tenant filtering if the requester is an operator